File: BCKEND/JAVA/BancoFinanceiro/src/Model/Cliente.java
package Model;

import java.util.Date;

public class Cliente {

	private String nome;
	private String cpf;
	private Date dataNasc;
	private Integer rg;
	private String email;
	private String telefone;
	private Endereco endereco;

	public Cliente(String nome, String cpf, Date dataNasc, Integer rg, String email, String telefone,
			Endereco endereco) {
		this.nome = nome;
		this.cpf = cpf;
		this.dataNasc = dataNasc;
		this.rg = rg;
		this.email = email;
		this.telefone = telefone;
		this.endereco = endereco;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getTelefone() {
		return telefone;
	}

	public void setTelefone(String telefone) {
		this.telefone = telefone;
	}

	public Endereco getEndereco() {
		return endereco;
	}

	public void setEndereco(Endereco endereco) {
		this.endereco = endereco;
	}

	public String getCpf() {
		return cpf;
	}

	public Date getDataNasc() {
		return dataNasc;
	}

	public Integer getRg() {
		return rg;
	}

	@Override
	public String toString() {
		return "Cliente " + nome + ", portador(a) do cpf " + cpf 
				+ ", nasceu em " + dataNasc.getDate() + "/" +
				dataNasc.getMonth() + "/" + dataNasc.getYear() 
				+ ", portador(a) do RG " + rg + ", " + email
				+ ", telefone: " + telefone + ", endereco: " 
				+ endereco;
	}

}

File: BCKEND/TypeScript/BancoFinanceiro/src/Model/Cliente.ts
import { Endereco } from "./Endereco";

export class Cliente {
    private nome: string;
    private cpf: string;
    private dataNascimento: Date;
    private rg: string;
    private email: string;
    private telefone: string;
    private endereco: Endereco;

    constructor(nome: string, cpf: string, dataNascimento: Date, rg: string, email: string, telefone: string, endereco: Endereco) {
        this.nome = nome;
        this.cpf = cpf;
        this.dataNascimento = dataNascimento;
        this.rg = rg;
        this.email = email;
        this.telefone = telefone;
        this.endereco = endereco;
    }

    public getNome(): string {
        return this.nome;
    }

    public getCpf(): string {
        return this.cpf;
    }

    public getDataNascimento(): Date {
        return this.dataNascimento;
    }

    public getRg(): string {
        return this.rg;
    }

    public getEmail(): string {
        return this.email;
    }

    public getTelefone(): string {
        return this.telefone;
    }

    public getEndereco(): Endereco {
        return this.endereco;
    }

    public setNome(nome: string): void {
        this.nome = nome;
    }

    public setEmail(email: string): void {
        this.email = email;
    }

    public setTelefone(telefone: string): void {
        this.telefone = telefone;
    }

    public setEndereco(endereco: Endereco): void {
        this.endereco = endereco;
    }

    public toString(): string {
        return `Nome: ${this.nome}
CPF: ${this.cpf}
Data de Nascimento: ${this.dataNascimento.toLocaleDateString('pt-BR')}
RG: ${this.rg}
Email: ${this.email}
Telefone: ${this.telefone}
Endereço: ${this.endereco.toString()}`;
    }

}